content/specific-skill-success-criteria/classes-and-objects/_index.md

```diff
@@ -1,6 +1,7 @@
 ---
 _db_id: 711
 content_type: project
+ready: true
 flavours:
 - any_language
 submission_type: link
```

Comment:
@sheenarbw I marked this as ready since it seems well specified to me but I'm not sure if you wanted some examples here, like the other items in `content/specific-skill-success-criteria`?

Reply:
It would be good to add examples. I'll make another issue for that though